src/Exceptions/TooManyHopsException.php

```diff
@@ -1,18 +1,14 @@
 <?php
 
+declare(strict_types=1);
+
 namespace Telkins\Dag\Exceptions;
 
 use Exception;
 
 class TooManyHopsException extends Exception
 {
-    /**
-     * Create a new exception instance.
-     *
-     * @param  int $maximumAllowableHops
-     * @return static
-     */
-    public static function make(int $maximumAllowableHops)
+    public static function make(int $maximumAllowableHops): TooManyHopsException
```

Comment:
I think the return type could/should be `self` rather than `TooManyHopsException`.